# Ts: 0.01 #sample_time
# pred_ts: 0.0 #预瞄时间，s
# matrix_q0: 0.05
# matrix_q1: 0.00
# matrix_q2: 0.1
# matrix_q3: 0.00
# max_iteration: 500
Ts: 0.01 #sample_time
pred_ts: 0.0 #预瞄时间，s
matrix_q0: 0.5
matrix_q1: 0.00
matrix_q2: 0.02
matrix_q3: 0.00

matrix_q4: 0.000
matrix_q5: 0.01
matrix_r0: 0.00
matrix_r1: 0.00
max_iteration: 500
# Ts: 0.02 #sample_time
# pred_ts: 0.0 #预瞄时间，s
# matrix_q0: 0.05
# matrix_q1: 0.01
# matrix_q2: 1.0
# matrix_q3: 0.01
# max_iteration: 500
